src/helpers/cliUtils.js

```diff
@@ -135,6 +135,7 @@ async function app(Client, fromDate, toDate, pathToConfig, pathToRunLogs, debug,
     if (debug) logger.level = 'debug';
     checkLogFile(pathToRunLogs);
     const config = getConfig(pathToConfig);
+    if (!config.commonExtractorArgs) config.commonExtractorArgs = {};
```

Comment:
DP - Same thinking here from my comment on the `ICARECSVClient.test.js` file; pretty sure this won't be needed if we make our `.implementation` check more robust. Let me know if I'm off-base.